### DISCUSSION TITLE
2.0.0 - Beta releases

### DISCUSSION THREAD

**Discussion by timohubois** [timber/timber#2724] [General]
It has been a few days since the last beta release for 2.0 and I would be interested to know if there are a new beta releases planned in the near future?

**Comment by rmens** [comment]
I would love to know too. With updates for PHP 8.0 ending this year and support for PHP 8.1 removed from Timber 1.x a beta or roadmap to the first 2.0 release would be highly appreciated.

**Comment by Levdbas** [comment]
Hi @timohubois and @rmens , the team is hard at work to make Timber 2.0 happen. We've just created an updated roadmap with open tickets, pull requests, etc. If you would like to help out on Timber, now is the perfect timing. See https://github.com/timber/timber/issues/2741

Closing this ticket in favor of the mentioned ticket above.
